ecs-init/docker/docker_test.go

```diff
@@ -94,7 +96,7 @@ func TestLoadImage(t *testing.T) {
 
 	mockDocker := NewMockdockerclient(mockCtrl)
 
-	mockDocker.EXPECT().LoadImage(godocker.LoadImageOptions{nil})
+	mockDocker.EXPECT().LoadImage(godocker.LoadImageOptions{nil, context.Background()})
```

Comment:
Looks like this should expect either `context.TODO()` or `gomock.Any()` instead of `context.Background()`.